core/src/main/java/io/crate/module/CrateCoreModule.java

```diff
@@ -53,6 +54,9 @@ public CrateCoreModule(Settings settings, IndexEventListenerProxy indexEventList
                 "but have not configured a license. Please request a license or deactivate the "+
                 "Enterprise Edition. https://crate.io/enterprise");
         }
+        if (CrateRestFilter.ES_API_ENABLED_SETTING.get(settings)) {
+            logger.warn("The Elasticsearch API is currently enabled, which makes it possible to request user data.");
```

Comment:
It's not possible to read `sys.users` without superuser privileges. It's also not possible if you have no user at all as with HTTP.